runtime/development/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "development-runtime"
-version = "0.10.46"
```

Comment:
We had this not updated, although I guess it's not a problem

Reply:
Unfortunately, this often falls short but it doesn't have any effect anyways.